LICENSE

```diff
@@ -1,3 +1,5 @@
+MIT License
```

Comment:
Do we need this line? Github seems to be able to figure out the licence without it.

Reply:
From the [Github docs](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository#determining-the-location-of-your-license):


> Most people place their license text in a file named LICENSE.txt (or LICENSE.md) in the root of the repository; here's an example from Hubot.
> 
> Some projects include information about their license in their README. For example, a project's README may include a note saying "This project is licensed under the terms of the MIT license."
> 
> As a best practice, we encourage you to include the license file with your project.

Reply:
I don't feel like that quote references having this one line in this license file?

Reply:
Oh, I read "Do we need this file", no we probably don't need it but I like it for clarity of the reader and have seen other projects include it

Reply:
ahh ok. That should be fine then, I haven't seen it myself so I wasn't sure.